run_dir/design/workset_samples.html

```diff
@@ -13,27 +13,78 @@
 <div id="page_content" style="display:none;">
 
     <h1>Workset <span id="page_title">{{ workset_name}}</span></h1>
-    <button type="button" class="btn btn-primary btn-lg" data-toggle="modal" data-target="#modal_plate">
+    <h3>Lims Step : <span id="span_lims_step"></span>, run on <span id="date_run"></span></h3>
```

Comment:
For these empty tags where we will be inserting data, I like to fill the tag with some holding content in the template. This makes it much more obvious when it has failed to load. Just a hyphen or question mark is fine. So for this case I would use:

```html
 <h3>Lims Step : <span id="span_lims_step">?</span>, run on <span id="date_run">?</span></h3>
```